package.json

```diff
@@ -2,46 +2,50 @@
   "name": "gulp-htmlmin",
   "description": "gulp plugin to minify HTML.",
   "version": "1.3.0",
-  "author": {
-    "name": "Jon Schlinkert",
-    "url": "https://github.com/jonschlinkert"
-  },
+  "homepage": "https://github.com/jonschlinkert/gulp-htmlmin",
+  "author": "Jon Schlinkert (https://github.com/jonschlinkert)",
   "contributors": [
     "Shinnosuke Watanabe <snnskwtnb@gmail.com> (https://github.com/shinnn)"
   ],
   "repository": "jonschlinkert/gulp-htmlmin",
+  "bugs": {
+    "url": "https://github.com/jonschlinkert/expand-files/issues"
+  },
   "license": "MIT",
-  "keywords": [
-    "format",
-    "gulp",
-    "gulpplugin",
-    "htm",
-    "html",
-    "minifier",
-    "minification",
-    "minify"
-  ],
   "files": [
     "index.js"
   ],
+  "main": "index.js",
```

Comment:
> I removed it from dev dependencies because it didn't work.

Interesting. Can you show the error log?

> In this project specifically, the reason the `main` property was added was that my linting tool detected the file in the root of the project, saw that it was missing from package.json and added it.

It's a bug of the linter. The entry point of a npm package is `index.js` by default, so the linter should not detect any errors in that case. Package authors don't need to explicitly specify `main` when the entry point is `index.js`.